tests/e2e-openshift/multi-cluster/create_otlp_receiver.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Get the cluster domain
+hostname_domain=apps.$(oc get dns cluster -o jsonpath='{.spec.baseDomain}')
+
+# Define the content for the collector
+collector_content=$(cat <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otlp-receiver
+  namespace: kuttl-multi-cluster-receive
+spec:
+  mode: "deployment"
+  ingress:
+    type: route
+    hostname: ${hostname_domain}
+    route:
+      termination: "passthrough"
+  volumes:
+    - name: kuttl-certs
+      configMap: 
+        name: kuttl-certs
+  volumeMounts:
+    - name: kuttl-certs
+      mountPath: /certs
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          http:
+            tls:
```

Comment:
Could we use edge termination or insecure routes? 

The recent version of the operator improved support for routes:
* hostname can be omitted
* the grpc route works for insecure and edge termination

Reply:
@pavolloffay I have updated the test case to remove setting the hostname. But for setting custom certs, ran into this issue. https://issues.redhat.com/browse/TRACING-3609 For the time being, I've kept the termination as passthrough and using the custom certs.